aoe_technology_radar-generateJson seems to require specific folder which does not exist by default

```
⋊> ~/S/p/tech-radar yarn aoe_technology_radar-generateJson                                                                                                                                          15:29:56
yarn run v1.22.17
$ /home/druf/Schreibtisch/projects/tech-radar/node_modules/.bin/aoe_technology_radar-generateJson
[ERROR] radar
ENOENT: no such file or directory, lstat '/home/druf/Schreibtisch/projects/tech-radar/radar'
[ERROR] radar
ENOENT: no such file or directory, lstat '/home/druf/Schreibtisch/projects/tech-radar/radar'
/home/druf/Schreibtisch/projects/tech-radar/build/rd.json created.
Done in 0.35s.
```

Maybe something is missing in the readme like `mkdir radar`.

--- a/dist_scripts/scripts/generateJson.js
+++ b/dist_scripts/scripts/generateJson.js
@@ -61,6 +61,7 @@ var __generator = (this && this.__generator) || function (thisArg, body) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 var fs = __importStar(require("fs-extra"));
+var file_1 = require("./generateJson/file");
 var paths = __importStar(require("./paths"));
 // Do this as the first thing so that any code reading it knows the right env.
 process.env.BABEL_ENV = "production";
@@ -73,6 +74,13 @@ process.on("unhandledRejection", function (err) {
 });
 fs.removeSync(paths.templateNodeModules);
 fs.ensureSymlinkSync(paths.appNodeModules, paths.templateNodeModules);
+try {
+    fs.statSync((0, file_1.radarPath)());
+}
+catch (e) {
+    console.error("".concat((0, file_1.radarPath)(), " not found. Please create ").concat((0, file_1.radarPath)(), " and add your markdown files to build the techradar."));
+    process.exit(1);
+}
 var generateJson = function () { return __awaiter(void 0, void 0, void 0, function () {
     var createRadar, save, radar;
     return __generator(this, function (_a) {

--- a/scripts/generateJson.ts
+++ b/scripts/generateJson.ts
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 import * as fs from "fs-extra";
 
+import { radarPath } from "./generateJson/file";
 import * as paths from "./paths";
 
 // Do this as the first thing so that any code reading it knows the right env.
@@ -16,6 +17,15 @@ process.on("unhandledRejection", (err) => {
 
 fs.removeSync(paths.templateNodeModules);
 fs.ensureSymlinkSync(paths.appNodeModules, paths.templateNodeModules);
+
+try {
+  fs.statSync(radarPath());
+} catch (e) {
+  console.error(
+    `${radarPath()} not found. Please create ${radarPath()} and add your markdown files to build the techradar.`
+  );
+  process.exit(1);
+}
 
 const generateJson = async () => {
   const { createRadar } = require("./generateJson/radar");

